cli/command/container/list.go

```diff
@@ -59,21 +59,12 @@ func newListCommand(dockerCli *command.DockerCli) *cobra.Command {
 	return &cmd
 }
 
-type preProcessor struct {
-	types.Container
```

Comment:
Maybe we can declare a custom type that happens to be backed by a map[string]bool?

```
// listOptionsProcessor is used to set any container list options which may only be embedded in the format template.
// This is passed directly into tmpl.Execute to in order to allow the preprocessor to set any list options that were not provided by flags (e.g. `.Size`).
// It is using a `map[string]bool` so that unknown fields passed into the template format do not cause errors. These errors will get picked up when running through the actual template processor.
type listOptionsProcessor map[string]bool
```